ckan/ckanext/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/resource_view.html

```diff
@@ -2,6 +2,13 @@
 
 {% block resource_view %}
   <div id="view-{{ resource_view['id'] }}" class="resource-view" data-id="{{ resource_view['id'] }}" data-title="{{ resource_view['title'] }}" data-description="{{ resource_view['descripion'] }}">
+    <a class="btn btn-secondary"
```

Comment:
`btn-secondary` doesn't exist in bs3, it should be `btn-default` until bootstrap upgrade is done.